src/NzbDrone.Core/Parser/ParsingService.cs

```diff
@@ -20,7 +20,10 @@ public interface IParsingService
         Series GetSeries(string title);
         RemoteEpisode Map(ParsedEpisodeInfo parsedEpisodeInfo, int tvdbId, int tvRageId, SearchCriteriaBase searchCriteria = null);
```

Comment:
Will Obsolete them. Removing them kills many parser tests. Save that for another "Parser Refactor" PR.